GWFish/modules/waveforms.py

```diff
@@ -519,51 +520,59 @@ def calculate_frequency_domain_strain(self):
     
         M = M1 + M2
         mu = M1 * M2 / M
-    
         Mc = cst.G * mu ** 0.6 * M ** 0.4 / cst.c ** 3
-    
-        # compute GW amplitudes (https://arxiv.org/pdf/2012.01350.pdf)
-        hp = cst.c / (2. * r) * np.sqrt(5. * np.pi / 24.) * Mc ** (5. / 6.) / \
-             (np.pi * ff) ** (7. / 6.) * (1. + np.cos(iota) ** 2.)
-        hc = cst.c / (2. * r) * np.sqrt(5. * np.pi / 24.) * Mc ** (5. / 6.) / \
-             (np.pi * ff) ** (7. / 6.) * 2. * np.cos(iota)
-    
+
+        ff = frequencyvector*cst.G*M/cst.c**3 #dimensionless frequency = f[Hz] * 4.926*10^{-6} * M[M_sol] 
+        ones = np.ones((len(ff), 1))
+
         C = 0.57721566  # Euler constant
         eta = mu / M
-    
-        f_isco = aux.fisco(self.gw_params)
+        eta2 = eta*eta
+        eta3 = eta2*eta
+
+        chi_eff = (M1*chi_1 + M2*chi_2)/M
+        chi_PN = chi_eff - 38/113*eta*(chi_1 + chi_2)
+        chi_s = 0.5*(chi_1 + chi_2)
+        chi_a = 0.5*(chi_1 - chi_2)
+
+        #f_cut = cut_order * f_isco
+        cut = self.gw_params['cut']
```

Comment:
Is this something that needs to be given by the user? I think we should at least a default (e.g. `cut = self.gw_params.get('cut', 4)`)

Reply:
Not what I meant, this still requires the user to go through the code and read the comment - I mean a default given in a way that one can _not specify_ `'cut'` as part of the `params` dictionary.

Reply:
I don't understand, sorry. The default value (4) is in gw_params and, in principle, one can not specify 'cut' as part of the parameters without having trouble.

Reply:
Ah! missed it, you are right.

Reply:
@AlessandroAgapito23 , around lines ≈507-590, you seemed to change `class TaylorF2` quite a lot. I do not understand what you did from looking at the changes, the code pieces that are added and removed do not correspond to each other. I would not change our standard TaylorF2 for the sake of reproducibility, so perhaps we should revert back to the old code. 

As I mentioned in the earlier comment, I would then break TaylorF2 and PhenomD into more functions and attributes, such that in `inspiral_corrections.py` we do not copy the code over from `waveforms.py`, instead we should only add new lines of code with PPE corrections.